src/actions/home/index.cr

```diff
@@ -1,5 +1,11 @@
 class Home::Index < BrowserAction
   get "/" do
-    render LuckyWeb::WelcomePage
+    wod = WodQuery.new.date(Time.now.at_beginning_of_day).first?
+
+    if wod.is_a?(Wod)
```

Comment:
I believe you can do `if wod` and that will check that it is not nil for you. Then you don't need the `is_a?`